### PR TITLE
Update

### PR DESCRIPTION
**Pull request by olafurpg**
the buildinfo setup is quite complicated, we could use something slightly more lightweight like `runMain("--classpath ...")` but I've found this buildinfo setup to be very useful in scalafix.

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.13
+sbt.version=0.13.16
```
Comment (xeno-by) [inline project/build.properties:1]: 1.0.0?

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,10 +1 @@
...
-// i.e. a storage for semantic information about Scala code.
-// See https://github.com/scalameta/scalameta/issues/605 for details.
-// The latest stable and pre-release version numbers can be found at http://scalameta.org/.
-addSbtPlugin("org.scalameta" % "sbt-scalahost" % "1.8.0")
-
-// sbt-coursier parallelizes downloads of dependencies.
-// If you haven't tried it yet, do so - the productivity boost it provides is insane.
-// It's especially useful for scala.meta, which contains more than a dozen of modules.
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC3")
+addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.7.0")
```
Comment (xeno-by) [inline project/plugins.sbt:1]: Why not coursier?

**Review by xeno-by — CHANGES_REQUESTED** [review]

File: project/plugins.sbt (hunk 2 above)
Comment (olafurpg) [reply project/plugins.sbt:1]: coursier is not necessary to use semanticdb.

File: project/build.properties (hunk 1 above)
Comment (olafurpg) [reply project/build.properties:1]: sbt 1.0 is not required to use semanticdb.

File: project/plugins.sbt (hunk 2 above)
Comment (xeno-by) [reply project/plugins.sbt:1]: Shouldn't we advertise it anyway? Downloading all the dependencies without it is super annoying.

File: project/build.properties (hunk 1 above)
Comment (xeno-by) [reply project/build.properties:1]: Fair enough.

File: project/plugins.sbt (hunk 2 above)
Comment (olafurpg) [reply project/plugins.sbt:1]: I think we should try to keep this template as minimal as possible, it should show the absolute minimum necessary to get started.

Comment (olafurpg) [reply project/plugins.sbt:1]: I feel like coursier is a distraction from that goal. The repo is so small that the download time is not a problem, this repo is not used for regular development.

**Review by xeno-by — APPROVED** [review]

**Pull request merged**